### PR TITLE
Jk/feature/user landing page

### LINKED ISSUE #19: User Landing Page
- [x] Includes greeting
- [x] Shows user data and allows for editing
- [x] Conditionally renders connection data
- [x] If no connection, display message and button for making connection
- [x] If connection, show connection name and about
- [ ] Connection section should include send postcard and end connection buttons

### PR DESCRIPTION
**Pull request by JeffKersting**
#### Purpose:
 > Check all that apply.
- [x] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [ ] Clean Up

### What does this PR do?
 > Clearly and concisely describe changes made. 

Adds a user landing page, with a user profile information section, and a user connection section. Conditionally renders to prompt a user to make a new connection if they have no existing connection.

Issues closed #19 

No problems, but the exact data flow is not nailed down on this iteration.

> Describe your code changes in detail for reviewers.

No changes, only additional component.

Added User-Landing.js and _landing.scss

N/A. Site navigation will need to be implemented prior to proper testing.

How do we want to display the address change form?

### CODE REVIEW

File: src/components/User-Landing-Page/User-Landing.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      <div className='info-wrapper'>
+        <h3>Address:</h3>
+        <h4>{user.address}</h4>
+        <h3>About:</h3>
+        <h4>{user.about}</h4>
+        <button>Edit</button>
+      </div>
+      <h2>Connection</h2>
+      <div className='info-wrapper'>
+        {!connection &&
```
Comment (kelsiebesingeryeh) [inline src/components/User-Landing-Page/User-Landing.js:27]: is this checking if the connection object is a falsey value? Is an empty object a falsey value or is it an array?

**Review by kelsiebesingeryeh — APPROVED** [review]

File: src/components/User-Landing-Page/User-Landing.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react'
+
+function UserLandingPage() {
+  const [user, setUser] = useState({id: 1, name: 'John', address: '123 Wherever St, Denver CO 80202', connections: false, about: 'My about me'})
```
Comment (gaj23) [inline src/components/User-Landing-Page/User-Landing.js:4]: Nice. I'm glad to have this hardcoded dummy data to start.

File: src/components/User-Landing-Page/User-Landing.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  const [connection, setConnection] = useState({id: 2, name:'Bill', about:'Howdy Im Bill'})
+
+  useEffect(() => {
+    console.log(user.connections)
+  }, [])
+
+  // PASS USER DATA THRU AS PROPS AFTER LOGIN, OR CALL THEM ON PAGE LOAD?
```
Comment (gaj23) [inline src/components/User-Landing-Page/User-Landing.js:11]: Good question. I liked the discussions we had, but I'm still not sure. I feel like I'm leaning towards passing down as props after login?

File: src/components/User-Landing-Page/User-Landing.js (hunk 1 above)
Comment (gaj23) [reply src/components/User-Landing-Page/User-Landing.js:27]: Good point. We could run a check by having a second, commented out `useState` for our connection set to the default of `null`. 
We might also want to confirm with the backend if a user starts with a connection or doesn't. I forget.

**Review by gaj23 — APPROVED** [review]
Thank you for these additions. I think we can incorporate the header you've used. It'll add nice juxtaposition to the blue we have through out.

**Pull request merged**